src/components/signup/Signup.css

```diff
@@ -0,0 +1,55 @@
+/* Signup */
+.signup-page {
+  display: flex;
+  flex-direction: row;
```

Comment:
direction is row by default. Remove this line unless you are trying to override a column direction